report/assets/styles.css

```diff
@@ -695,11 +695,10 @@
   border-top: 1px solid var(--report-border-color-secondary);
 }
 
-.lh-metric:nth-last-child(-n+2) {
+.lh-category:not(.lh--hoisted-meta) .lh-metric:nth-last-child(-n+2) {
   border-bottom: 1px solid var(--report-border-color-secondary);
```

Comment:
was thinking about how else to do this, and a `:not(:last-child) ...` almost works but `lh-metrics-container` isn't the last element normally anyways so nope.